src/components/GameBoard.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState, useEffect } from 'react';
+import { Card } from './Card'
+import '../styles/gameBoard.scss'
+import { v4 as uuidv4 } from 'uuid';
+
+const cardPool = [
+    {
+        type: 'faceGrin',
+        image: '😃'
+    },
+    {
+        type: 'faceSquint',
+        image: '😆'
+    },
+    {
+        type: 'animalOrangutan',
+        image: '🦧'
+    },
+    {
+        type: 'faceHalo',
+        image: '😇'
+    },
+    {
+        type: 'faceWink',
+        image: '😉'
+    },
+    {
+        type: 'animalGorilla',
+        image: '🦍'
+    },
+    {
+        type: 'animalPoodle',
+        image: '🐩'
+    }
+    ,
+    {
+        type: 'animalMouse',
+        image: '🐭'
+    }
+];
+
+const createShuffledDeck = deck => {
+    deck = [...deck, ...deck];
+    let count = deck.length;
+    while (count > 0) {
+        let randomPick = Math.floor(Math.random() * count);
+        count--;
+        [deck[count], deck[randomPick]] = [deck[randomPick], deck[count]];
+    }
+    return deck.map(each => ({ ...each, 'id': uuidv4() }));
```

Comment:
1. You can just use `card` instead because what you are mapping through is eventually a `card` object. Having an `each` in front it's un-needed.
2. Sounds like a plan. 👍 Normally you would want function to only do one thing but with proper naming at least the user  will know what exatly is the function doing.